unlock-protocol.com/blog/ethwaterloo-tickets.md

```diff
@@ -0,0 +1,56 @@
+---
+title: EthWaterloo Tickets
+subTitle: We're proud to announce that EthGlobal is using Unlock's ticket application for its EthWaterloo hackathon, for crypto and non crypto users alike!
+authorName: Julien Genestoux
+publishDate: October 2, 2019
+description: We're proud to announce that EthGlobal is using Unlock's ticket application for its EthWaterloo hackathon, for crypto and non crypto users alike!
+image: /static/images/blog/ethwaterloo-tickets/ethwaterloo.png
+---
+
+[EthGlobal](https://ethglobal.co/) runs Ethereum hackathons all around the world for people to learn and practice their Ethereum coding skills. One of their goals is to "onboard" hackers and engineers who are not yet familiar with Ethereum and don't already have a wallet or Ether. Yet, they rightfully live by their words and insist on using [Ethereum powered tickets](https://unlock-protocol.com/blog/introducing-tickets/)! By adding [user accounts](https://unlock-protocol.com/blog/unlock-user-accounts/) to our tickets app, Unlock is excited to deliver a great on-boarding experience! Here is how it works:
+
+1. A hacker who's never used Ethereum before applied and was accepted for [EthWaterloo](https://ethwaterloo.com/)!
+2. She's sent to our [registration page](https://bit.ly/2ozn9iI). Note: do NOT purchase a ticket if you have not been previously approved. Your funds will be lost and you will not be able to attend.
+
+![Event Page](/static/images/blog/ethwaterloo-tickets/event-page.png)
+
+3. When clicking on the registration button, shes' asked to pay 20 DAI which she does not have because she does not have a wallet. But, when clicking on the Purchase button, she's asked to login or sign up.
```

Comment:
```suggestion
3. When clicking on the registration button, she's asked to pay 20 DAI which she does not have because she does not have a wallet. But, when clicking on the Purchase button, she's asked to login or sign up.
```

Reply:
Thx!